amis/packer_centos7.json

```diff
@@ -10,6 +10,7 @@
     "build_date" : "{{env `BUILD_DATE`}}",
     "vendor_path" : "{{env `VENDOR_PATH`}}",
     "nvidia_enabled" : "{{env `NVIDIA_ENABLED`}}",
+    "dcv_installed": "{{env `DCV_INSTALLED`}}",
```

Comment:
please add this var into the "guide" of the `build_ami.sh` script

Reply:
added